### PR TITLE
bump gomodules-extracted

### PR DESCRIPTION
**Pull request by hensur**
depends on https://github.com/dependabot/gomodules-extracted/pull/6

This should resolve some issues I had with go modules in gitlab subgroups: https://github.com/dependabot/dependabot-core/issues/2618

### CODE REVIEW

**Comment by jurre** [comment]
Thanks @hensur, I wonder, now that we're pulling in `golang.org/x/mod` (which wasn't available previously I believe) do we even need the `gomodules-extracted` dependency? Would be great to just rely on the native package

**Comment by hensur** [comment]
I noticed this as well and checked if we can import all go mod dependencies from `golang.org/x/mod`. But there are still some functions in internal packages that we have to import from `gomodules-extracted`. Take a look at the imports in `go_modules/helpers/updatechecker/main.go`. I couldn't find a exported version of the internal imports in `golang.org/x/mod`.

File: go_modules/helpers/go.sum [hunk 1]
```diff
@@ -4,3 +4,22 @@ github.com/dependabot/gomodules-extracted v0.0.0-20181020215834-1b2f850478a3 h1:
 github.com/dependabot/gomodules-extracted v0.0.0-20181020215834-1b2f850478a3/go.mod h1:+dRXSrUymjpT4yzKtn1QmeknT1S/yAHRr35en18dHp8=
 github.com/dependabot/gomodules-extracted v1.1.0 h1:k0Fcin3JZBfd/yW0JEeg38UtTiZl6E9uYkdqA4Dqbyw=
 github.com/dependabot/gomodules-extracted v1.1.0/go.mod h1:+dRXSrUymjpT4yzKtn1QmeknT1S/yAHRr35en18dHp8=
```
Comment (jurre) [inline go_modules/helpers/go.sum:6]: Would you mind running `go mod tidy` in this dir?

Comment (hensur) [reply go_modules/helpers/go.sum]: done!

**Pull request merged**